src/Themes/DigitalTheme/Views/Dividers.liquid

```diff
@@ -1,14 +1,13 @@
-{% assign divider = Model.Divider %}
+{% assign selectedValues = Model.SelectedValues %}
```

Comment:
I think these should be modified outside of this pr. and migrate the content types as well.